program/src/processor/sweep_fees.rs

```diff
@@ -119,6 +154,7 @@ pub(crate) fn process(program_id: &Pubkey, accounts: &[AccountInfo]) -> ProgramR
     )?;
 
     market_state.accumulated_fees = 0;
+    market_state.accumulated_royalties = 0;
 
```

Comment:
What do you think of also updating market_state.royalties_bps here according to the metadata account (to account for changes to the metadata since the market was initialized)?

Reply:
Note - would need to ensure that this plays nicely in the scenario in which a new_order occurs, then royalties_bps is updated, then consume_events occurs.